packages/gamut/src/Markdown/__tests__/Markdown.test.tsx

```diff
@@ -167,86 +179,88 @@ var test = true;
         },
         code: {
           component: (props: React.HTMLProps<HTMLElement>) => (
-            <strong {...props} />
+            <strong {...props}>custom code snippet 1</strong>
           ),
         },
       };
 
-      const { wrapper } = renderWrapper({ text, overrides });
-
-      expect(wrapper.find(overrides.CodeBlock.component).length).toEqual(1);
-      // There should only be one <code /> override because the codeblock override overwrote it
-      expect(wrapper.find(overrides.code.component).length).toEqual(1);
+      const { view } = renderView({ text, overrides });
+      view.getByText('custom codeblock 2');
+      view.getByText('custom code snippet 1');
     });
   });
 
   it('Renders data attributes on the markdown wrapper', () => {
-    const { wrapper } = renderWrapper({
+    renderView({
       text: basicMarkdown,
       'data-testid': 'cool',
     } as any);
 
-    expect(wrapper.find('div[data-testid="cool"]').length).toEqual(1);
+    expect(document.querySelectorAll('div[data-testid="cool"]').length).toEqual(
+      1
+    );
```

Comment:
this can be simplified to
```suggestion
view.getByTestId('cool')
```